code/modules/mob/living/carbon/emote.dm

```diff
@@ -11,7 +11,29 @@
 	key_third_person = "blinks"
 	message = "blinks."
 
-/datum/emote/living/carbon/blink_r
+/datum/emote/living/carbon/blink/run_emote(mob/living/carbon/human, params, type_override, intentional)
+	. = ..()
+
+	// Don't blink. Not even for a second.
+	for(var/obj/structure/statue/peanut in view(8, human))
+		if (prob(99))
+			// Not feeling it right now...
+			continue
+
+		// Security guards go here to die.
+		if(do_teleport(peanut, human, channel = TELEPORT_CHANNEL_MAGIC))
+			human.log_message("blinked and had their neck snapped by [peanut]!")
+			human.visible_message(span_danger("[human] blinks, and [peanut] snaps [human.p_their()] neck in a flash!"), \
+				span_userdanger("The instant you close your eyes, [peanut] snaps your neck!"))
+
+			var/obj/item/bodypart/head/head = human.get_bodypart(BODY_ZONE_HEAD)
+			head.receive_damage(brute=200)
```

Comment:
runtimes if the human has no head lol lmao dullahans

Reply:
god why cant people just have heads